conf/application.conf.template

```diff
@@ -36,6 +36,7 @@ play {
     ws.timeout.connection              =   10000
     filters.headers.contentSecurityPolicy = "default-src 'self'; script-src 'self' '%NONCE-SOURCE%' https://forums.spongepowered.org https://forums-cdn.spongepowered.org https://www.google-analytics.com/analytics.js; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://forums-cdn.spongepowered.org; img-src 'self' https:; font-src 'self' https://fonts.gstatic.com; frame-src https://forums.spongepowered.org; frame-ancestors 'none'; block-all-mixed-content"
```

Comment:
is this not obsolete with this PR?

Reply:
it is still being applied, but the new filter will overwrite the header. I kept it for reference.

Reply:
@pschichtel I wanted to modify something in the CSP and I found out that this is not going to be overriden. Maybe because we switched to play 2.6?

Reply:
The code this is based on is actually from a play 2.6. Has something changed on your configuration? (especially filter settings)

Reply:
It's the config in the master branch. So I don't think so

Reply:
can you check if it is even getting applied? Maybe it was wrongly migrated to 2.6? The configuration stuff had quite a few changes. Now that you are on 2.6 I could PR the exact version from my project.

Reply:
I'm not sure if it's applied. If you want to go for the PR.